tests/bsim/bluetooth/mesh/src/test_brg.c

```diff
@@ -246,22 +246,26 @@ static void bridge_entry_remove(uint16_t src, uint16_t dst, uint16_t net_idx1, u
 static void tester_bridge_configure(void)
 {
 	uint8_t status;
-	int err;
 
 	LOG_INF("Configuring bridge...");
 
+	ASSERT_OK(
+		bt_mesh_cfg_cli_net_transmit_set(0, BRIDGE_ADDR, BT_MESH_TRANSMIT(2, 20), &status));
+	if (status != BT_MESH_TRANSMIT(2, 20)) {
+		FAIL("Net transmit set failed (status %u)", status);
+	}
+
```

Comment:
This seems unrelated to bringing TC support back?

Reply:
Nop. When TC is back, some collisions happen. This change adds more transmission attempts to solve this. The change doesn't have sense without TC.